pkg/migrations/op_common_test.go

```diff
@@ -40,6 +44,14 @@ func ExecuteTests(t *testing.T, tests TestCases, opts ...roll.Option) {
 	testSchema := testutils.TestSchema()
 
 	for _, tt := range tests {
+		if tt.minPgMajorVersion != 0 && os.Getenv("POSTGRES_VERSION") != "" {
```

Comment:
I had to add this because setting a column list for `ON DELETE` actions was released in pg15.